src/Status.js

```diff
@@ -2,7 +2,6 @@ function updateTaskStatus(taskIndex, status, tasksLocal) {
   tasksLocal[taskIndex - 1].completed = status;
   localStorage.setItem('tasks', JSON.stringify(tasksLocal));
 }
-
 function clearCompletedTasks(tasksLocal) {
   const updatedTasks = tasksLocal.filter((task) => !task.completed)
     .map((task, index) => ({
```

Comment:
- [ ] Great job implementing clearAllCompleted function to clear all the completed tasks. However, there's a small bug here. The app does not behave consistently. When I added three todos and marked one as completed. All the todos cleared off (both the completed and uncompleted) when I clicked the `clear all completed` button. Take a look at below screenshot.

### Error One
[todo-error.webm](https://github.com/Rdzaman20000/TodoList-V2/assets/26597432/f16bc0c7-1db1-4619-a071-1ed55fb8de48)

- [ ] Another unexpected behaviour of your app is that, when I added three tasks and I deleted one. When I tried to add a new task, it override the previous two and one the new task displayed on the browser and in the local storage. Have a look :point_down: 

### Error Two
[app-behave.webm](https://github.com/Rdzaman20000/TodoList-V2/assets/26597432/8f54f4c9-d547-49fd-8163-27834a4a2db2)